docs/en/stack/ml/anomaly-detection/ootb-ml-jobs-apm.asciidoc

```diff
@@ -80,15 +80,17 @@ This job is useful in identifying bots.
 [[apm-transaction-jobs]]
 == Transactions
 // tag::apm-transaction-jobs[]
-Detect anomalies in transactions from your APM services. These configurations 
-are only available if data exists that matches the following term query: 
-`"processor.event": "transaction"`.
+Detect anomalies in transactions from your APM services for metric data. These 
+configurations are only available if data exists that matches the following term 
+queries: `"processor.event": "metric"` and `"metrcset.name": "transaction"`.
 
-high_mean_transaction_duration::
+apm_metrics::
 
-* For transaction data where `processor.event` is `transaction`.
-* Models duration of transactions by transaction type for APM services.
-* Detects anomalies in high mean of transaction duration.
+* For metric data where `processor.event` is `metric`.
```

Comment:
For completeness, you should say where `processor.event` is `metric` and `metricset.name` is `transaction`

Reply:
Completed in c348ee5.